### DISCUSSION TITLE
Press reports

### DISCUSSION THREAD

**Discussion by MikeMcC399** [corona-warn-app/cwa-website#3556] [General]
Posting here just in case the app gets reactivated out of its "sleep mode" any time.

https://www.n-tv.de/panorama/Bund-steckte-214-Millionen-Euro-in-Corona-Warn-App-article24361420.html

> Warnung vor Kontaktpersonen, Impfstatus, Infoquelle: Fast 49 Millionen Menschen in Deutschland nutzen die Corona-Warn-App während der Pandemie. Die kostet den Bund auch ordentlich Geld. Einer Simulation zufolge verhindert sie aber auch zahlreiche Todesfälle.
> ...

**Comment by Ein-Tim** [comment]
ntv mainly uses this Bundestagsdrucksache https://dserver.bundestag.de/btd/20/081/2008108.pdf as source.
